apollo-adminservice/src/main/java/com/ctrip/framework/apollo/adminservice/AdminServerEurekaClientConfigure.java

```diff
@@ -0,0 +1,42 @@
+package com.ctrip.framework.apollo.adminservice;
+
+import com.ctrip.framework.apollo.biz.eureka.ApolloEurekaClientConfig;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.cloud.context.scope.refresh.RefreshScope;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.event.EventListener;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * @author : kl
+ * After startup, set FetchRegistry to true, refresh eureka client
+ **/
+@Configuration
+@ConditionalOnProperty(value = {"eureka.client.enabled"}, havingValue = "true", matchIfMissing = true)
+public class AdminServerEurekaClientConfigure {
```

Comment:
It looks like that `AdminServerEurekaClientConfigure` and `ConfigServerEurekaClientConfigure` are exactly the same, is it better to move them to package `com.ctrip.framework.apollo.biz.eureka` so that we could avoid duplication?

Reply:
agree